pkg/controller/node/node_controller.go

```diff
@@ -800,6 +809,30 @@ func (ctrl *Controller) getNodesForPool(pool *mcfgv1.MachineConfigPool) ([]*core
 	return nodes, nil
 }
 
+// setClusterConfigAnnotation reads cluster configs set into controllerConfig
+// and add/updates required annotation to node such as ControlPlaneTopology
+// from infrastrcture object.
+func (ctrl *Controller) setClusterConfigAnnotation(nodes []*corev1.Node) error {
+	cc, err := ctrl.ccLister.Get(ctrlcommon.ControllerConfigName)
+	if err != nil {
+		return err
+	}
+
+	for _, node := range nodes {
+		glog.Infof("Initiating controlPlaneTopology annotation %s\n", string(cc.Spec.Infra.Status.ControlPlaneTopology))
+		if node.Annotations[daemonconsts.ClusterControlPlaneTopologyAnnotationKey] != string(cc.Spec.Infra.Status.ControlPlaneTopology) {
```

Comment:
It will always get set and would have default value HighlyAvailable, see https://github.com/openshift/api/blob/master/config/v1/0000_10_config-operator_01_infrastructure.crd.yaml#L190